File: Kotlin/entradaSaida/Exercicio2.kt
package entradaSaida

fun main(){
    val banhoDezMin = 10
    var litrosPorMin = 120
    litrosPorMin /= banhoDezMin
    print("Digite a quantidade de minutos do banho: ")
    val qtdMinutos = readln().toInt()

    val totalLitros = qtdMinutos * litrosPorMin

    print("Um banho de $qtdMinutos gasta $totalLitros l")
}

File: Ruby_/entradaSaida/Exercicio2.rb
# frozen_string_literal: true

banhoDez = 10
lPorMin = 120
lPorMin /= banhoDez

print "Digite a quantidade de minutos do banho: "
qtdMinutos = gets.to_i

totalLitros = qtdMinutos * lPorMin

puts "Um banho de #{qtdMinutos} minutos gasta #{totalLitros}l"